src/encoded/static/components/dataset.js

```diff
@@ -1206,28 +1207,92 @@ FilePanelHeader.propTypes = {
 };
 
 
-function displayPossibleControls(item, adminUser) {
-    if (item.possible_controls && item.possible_controls.length > 0) {
-        return (
+/**
+ * Series experiment table target column definition.
+ */
+const seriesTableTargetColumn = {
```

Comment:
This reminds me of Javascript code in a pre-Class word where objects were used to encapsulate functionality like a Class could. Unless you are concerned with performance, would as class suffice?